go/mysql/query.go

```diff
@@ -590,6 +600,38 @@ func (c *Conn) writeFields(result *sqltypes.Result) error {
 	return nil
 }
 
+// writeFieldsList writes the fields of a Result. It should be called only
+// if there are valid columns in the result.
+func (c *Conn) writeFieldsList(result *sqltypes.Result, wilds []string) error {
+	// Now send each Field.
+	for _, field := range result.Fields {
+		for _, wild := range wilds {
+			if field.Name == wild {
+				field.Database = strings.TrimPrefix(field.Database, "vt_")
```

Comment:
this seems unnecessary. it's only really going to be used by a proper mysql client, at which point we should return exactly what it asks for